dispatch-build-bottle/action.yml

```diff
@@ -0,0 +1,124 @@
+name: Dispatch build bottle
+inputs:
+  formula:
+    description: "Formula name"
+    required: true
+  issue:
+    description: "Issue number, where comment on failure would be posted"
+    required: true
+  ignore-errors:
+    description: "Ignore errors (default: false)"
+    required: false
+  macos:
+    description: "macOS Version"
+    required: false
+  name:
+    description: "Name"
+    required: false
+  sender:
+    description: "Sender"
+    required: true
+  upload:
+    description: "Whether to upload built bottles or not (default: false)"
+    required: false
+runs:
+  using: "composite"
+  steps:
+    - name: ${{ inputs.formula }}
+      id: print_details
+      run: |
+        echo formula=${{ inputs.formula }}
+        echo sender=${{ inputs.sender }}
+        echo name=${{ inputs.name }}
+        echo ignore-errors=${{ inputs.ignore_errors }}
+        echo version=${{ inputs.macos }}
+        echo issue=${{ inputs.issue }}
+        echo upload=${{ inputs.upload }}
+        echo "::set-output name=repo::$(brew --repo ${{github.repository}})"
+
+    - name: Set up Homebrew
+      id: set-up-homebrew
+      uses: Homebrew/actions/setup-homebrew@master
+
+    - name: Run brew test-bot --only-cleanup-before
+      if: github.repository == 'Homebrew/homebrew-core'
+      id: test-bot-only-cleanup-before
+      run: brew test-bot --only-cleanup-before
```

Comment:
Steps which have  `if: github.repository == 'Homebrew/homebrew-core'`, were present only in `homebrew-core`'s `dispatch-build-bottle` action,  but not for `linuxbrew-core` 

Can this condition be modified to `if: always()` ?  I'm not sure and need input here

Reply:
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;No, some users might (well?) then run into issues like the ones I and @reitermarkus fixed in Homebrew/brew#9457, Homebrew/homebrew-core#65917, and Homebrew/homebrew-cask#89714.  You can still loosen this condition to only run actions if they're triggered from within the Homebrew organization, though.